README.md

```diff
@@ -11,4 +11,7 @@ So here it is, the project that will make it easier to keep up to date with devR
 - When making a new meetup the creator can say in which country it will be. Other people can subscribe to a country to get notifications.  
 _Example: Im makimg a new meetup for the Netherlands, if there is a few belgians, french people and germans willing to join the location will average to some place in Limburg probably._
 - Sharing nice places, some meetups might be in a park and if that was easily accessible and has enough park benches it might become a recommended place which weighs in the location averager.
+- making this into a platform that can be used outside of devrant too
```

Comment:
Capitalize the first word of a sentence :P

Reply:
mierenneuker :P maar je hebt gelijk, natuurlijk